### ISSUE TITLE
Stopped working?

### ISSUE THREAD

**Issue by Christo44** [mrmlnc/vscode-scss#42]
Hi, this plugin is or was a timesaver and wow what a bonus tool, but for some reason it stopped working.

It just does not read my variables anymore.  I'm going to disable all my plugins and check again.

**Comment by mrmlnc** [comment]
Hello, @Christo44,

What about small code sample?

**Comment by Christo44** [comment]
Hi, 

not sure if this will suffice, there is just no completion.

I created two files:
```
[scss] (root folder)
|- main.scss
|--[vendor]
    |-- type.scss
```
in type.scss
```
$text-dark: black;
```
so in main.scss:
```
@import "type"
body {
   color: "does not complete the variables here"
}
```

so after doing this, I removed my settings, and all modules, then added back one by one. 

Now it works again.

Go figure..

**Comment by Christo44** [comment]
Actually it stopped working again

**Comment by mrmlnc** [comment]
Well, IMHO, this is problem of my plugin where any of files have bad syntax or other problem with parsing. 

Now you can reload your workspace by reload command from F1 menu.

 I'll work on common  `vscode-css-intelliservice` (for CSS and preprocessors) soon and resolve this issue. 

Sorry for this situation, but now I have no free time :(

**Comment by Christo44** [comment]
Maybe it's another plugin causing a clash ?

For what it's worth here's what happens when I start typing a variable name:
https://dl.dropboxusercontent.com/u/5386287/git/screenshot.png

You'll notice there are names in the list, just not my variables.

**Comment by Christo44** [comment]
I know you don't have time, but this shows a basic example, for what it's worth:

https://dl.dropboxusercontent.com/u/5386287/git/no-suggestions.png

**Comment by Christo44** [comment]
This is interesting.

I remove:
`@include font-face("Akkurat-Bold", "../fonts/akkurat/lineto-akkurat-bold");` from my base.scss file, and the completions are back..
that's calling bourbons font-face mixin.

**Comment by mrmlnc** [comment]
Well, I tried to reproduce your issue but I couldn't do it. You can create demo repository or ZIP/TAR archive?

![extension development host - main scss - root_files 2017-06-03 13-08-46](https://cloud.githubusercontent.com/assets/7034281/26752662/d0084fd0-485d-11e7-9610-d8a86af9f4ae.png)

**Comment by Christo44** [comment]
Hi, thanks
Ok will have a look and share my code with you first thing in the morning

**Comment by Christo44** [comment]
Hi, 
at first I thought, yours works because you're not using underscore scss files.  In my example though I was using

I made you a example zip and I found out some interesting things, first everything works in the zip, even using underscores.

For what it's worth, in my zip example (didn't load it cause it's just like yours only with underscores..)
First I imported like this. Note the filename is actually __bourbon.scss_

`@import "./vendor/bourbon/bourbon";`

that of course gives me an error:
```
[Error - 8:46:51 PM] (node:7877) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: ENOENT: no such file or directory, stat 'path-to-folder/source/assets/scss/vendor/bourbon/bourbon.scss
```

then I switch the import path too:
`@import "./vendor/bourbon/_bourbon";`

Which gives me no errors anymore of course.

On my bigger project however it's not working even with the paths corrected. I got it working by removing @import in my _base.scss file containing most of the variables, but obviously that's not a definitive answer, it could be that it works by coincidence... so ya.  I found that scss-scan works but again it's not as feature loaded as I like in your plugin.

**Comment by mrmlnc** [comment]
Thanks for your detailed answer! I really appreciate it 👍 

 I'll try to find time to solve your problem.

**Comment by Christo44** [comment]
Cool beans, it's such a great plugin.

I was thinking I could replace all the underscores in my current project, and update my gulp file to test again. I'm going to try do that on monday.

**Comment by Christo44** [comment]
Hey, 

did you get my email ?

Also not sure but I might have found a workaround. I added all the `@import "_filename.scss"` into `"all.scss"` and called it once, and looks like that might have worked. No errors no `Error: ENOENT: no such file or directory, stat 'path-to-folder/source/assets/scss/vendor/bourbon/bourbon.scss`

**Comment by raldred** [comment]
Intellisense has stopped working for me too.
My project is growing quickly so it's possible it's something i've introduced as it was working.
I don't get any errors from the extension in dev tools.

It did stop working around the latest VSCode 1.13.0 update, this may just be a coincidence

**Comment by mrmlnc** [comment]
Hello, @Christo44,

 Sorry for delay. I read your Email, but but forgot to reply. You can use https://www.bountysource.com/trackers/46510797-mrmlnc-vscode-scss or https://paypal.me/mrmlnc

**Comment by mrmlnc** [comment]
Okay, now I can offer you to try the new version of the extension. Now is beta, but I need feedback.

https://github.com/mrmlnc/vscode-scss/releases/tag/0.7.0

How to install:

  * Download `.vsix` file from release page
  * Open VS Code
  * F1 → Install extension from VSIX
  * Reload editor by button after install

**Comment by mrmlnc** [comment]
Closed in favor of #49
